tools/run_session

```diff
@@ -141,8 +140,9 @@ fi
 # tutorial and main missions.
 sleep 6
 
-# Get rid of any pre-existing discrete events JSON files.
+# Get rid of any pre-existing discrete events and self-reports JSON files.
 $rm -f "${TOMCAT}"/external/malmo/Minecraft/run/saves/discrete_events/discrete_events.json
+rm -f "${TOMCAT}"/external/malmo/Minecraft/run/saves/self_reports/*.json
```

Comment:
@paulosoaresua can you change this back to `$rm` or to `/bin/rm`? On some computers (e.g. Emily's) `rm` is aliased to something else.

Reply:
Oh, I din't realize that. Issue fixed.

Reply:
@adarshp, I cannot see the details of the error in the test.

Reply:
Not sure what happened - the MacOS test timed out- I’ll merge it.

Reply:
Ok. I ran it in my machine with no problems.